spec/core/v2/ics-004-packet-semantics/README.md

```diff
@@ -190,10 +167,23 @@ The architecture of clients, connections, channels and packets:
 
 #### Store paths
 
+NOTE do we stil need this, or we can retrieve the sequence from the commitment path associated with the clientID? 
+
+- The `nextSequenceSend`, stored separately, tracks the sequence number for the next packet to be sent.
+
+```typescript
+function nextSequenceSendPath(sourceID: bytes, destID: bytes): Path {
```

Comment:
This shouldn't be keyed on the string `clients`.

And there's no need for there to be both source and destId. Since we only need our sides' identifier to uniquely key on the channel.

It can just be

`nextSequenceSend/{sourceId}`